webapp-mgmt/cas-management-webapp-configuration/src/main/java/org/apereo/cas/mgmt/configuration/CasManagementConfigurationProperties.java

```diff
@@ -0,0 +1,85 @@
+package org.apereo.cas.mgmt.configuration;
+
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.configuration.model.core.CasServerProperties;
+import org.apereo.cas.configuration.model.core.authentication.AuthenticationProperties;
+import org.apereo.cas.configuration.model.core.services.ServiceRegistryProperties;
+import org.apereo.cas.configuration.model.webapp.LocaleProperties;
+import org.apereo.cas.mgmt.configuration.model.ManagementWebappProperties;
+import org.springframework.boot.autoconfigure.web.ServerProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+@ConfigurationProperties("cas")
+public class CasManagementConfigurationProperties {
+    /**
+       Authentication
+     */
+    @NestedConfigurationProperty
+    AuthenticationProperties authn = new AuthenticationProperties();
```

Comment:
Are these settings/properties used anywhere here in the management app?

Reply:
Yes, I believe mostly by the ldap module

Reply:
Hmm...confused here! Why would the management app want to deal with anything authentication-related? Are you referring to ways one get to the management app? I dont recall doing anything ever in the management.properties file that would take the form of `cas.authn.something`. Am I missing something here?

Reply:
It is used here:
``` @RefreshScope
    @ConditionalOnMissingBean(name = "attributeRepository")
    @Bean
    public IPersonAttributeDao attributeRepository() {
        return Beans.newStubAttributeRepository(casProperties.getAuthn().getAttributeRepository());
    }
```
and Here
```
    @RefreshScope
    @Bean
    public Collection<BaseOidcScopeAttributeReleasePolicy> userDefinedScopeBasedAttributeReleasePolicies() {
        final OidcProperties oidc = casProperties.getAuthn().getOidc();
        return oidc.getUserDefinedScopes().entrySet()
                .stream()
                .map(k -> new OidcCustomScopeAttributeReleasePolicy(k.getKey(), CollectionUtils.wrapList(k.getValue().split(","))))
                .collect(Collectors.toSet());
    }
```

Reply:
Ah, gotcha. 

So, mostly suggestions then:

1. So the stub attribute repository, that just needs a map right? I wonder if we can remove the reference in the ctor and simply introduce a new setting in the `ManagementProperties` file. Don't know. This feels like the wrong place to define attributes used in the UI only, under an "authn" category. 

2. Same thing with the OIDC user scoped. All we need is a key and value, right? How about we just define a map directly and forego the `OidcProperties`.

This is perfectly good as is; I am mostly nit-picking really.